addons/website_sale/views/templates.xml

```diff
@@ -1195,7 +1195,7 @@
             <t t-set="additional_title">Shop - Checkout</t>
             <t t-set="no_footer" t-value="1"/>
             <div id="wrap">
-                <div class="container oe_website_sale py-2">
+                <div class="container oe_website_sale pt-2 pb-5">
```

Comment:
Not sure: should we avoid removing a class in stable version? Maybe you can just add a css rule on oe_website_sale, with a comment for explanation... actually if we only do this pt-2 pb-5, in one year this is gone since there is no explanation.

Reply:
I do agree, so something as it was initially done on https://github.com/odoo/odoo/pull/67732

Reply:
Yes, with still the concerns I have written on JKE's PR -> to test, that was _guesses_ of potential issues

Reply:
Actually I see that we had already solved this problem [in some place](https://github.com/odoo/odoo/blob/14.0/addons/website_sale/views/templates.xml#L1440) by adding a margin bottom on the buttons.
So adding a rule on a parent element ('oe_website_sale' in no footer for exemple) will increase the margin in places where it should not be.
So for me the solution would be to do the same thing (add mb32 on the buttons) where it is needed.

Reply:
@qsm-odoo concern `actually if we only do this pt-2 pb-5, in one year this is gone since there is no explanation.` still remain, I prefered jke solution with a custom class that no one will dare removing without understanding what it does.

I am ok with whatever, as long as the user can click on the button, especially in modals (which impacts every module).